src/wrappers.js

```diff
@@ -15,17 +15,16 @@ import {
 
 import { reducer as userReducer, name as userSliceName } from "./features/user"
 
-import {
-  reducer as databaseReducer,
-  name as databaseSliceName,
-} from "./features/database"
-
 import {
```

Comment:
**suggestion:** Delete all of this database stuff from the code base as it's no longer used...